lib/helpers/helper.theme.ts

```diff
@@ -0,0 +1,7 @@
+export const addDarkMode = () => {
+  document.body.classList.add("dark-mode")
+}
+
+export const removeDarkMode = () => {
+  document.body.classList.remove("dark-mode")
+}
```

Comment:
I think they should be called:
`useLightMode` and `useDarkMode`.
The fact that the function is toggling the `dark-mode` class is an implementation detail.